app/code/Bulbulatory/Recommendations/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="bulbulatory" translate="label" sortOrder="10">
+            <label>Bulbulatory</label>
+        </tab>
+        <section id="recommendations" translate="label" type="text" sortOrder="130" showInDefault="1" showInWebsite="1" showInStore="1">
+            <class>separator-top</class>
+            <label>Recommendations</label>
+            <tab>bulbulatory</tab>
+            <resource>Bulbulatory_Recommendations::recommendations_config</resource>
+            <group id="general" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
+                <label>General Configuration</label>
+                <field id="enable" translate="label" type="select" sortOrder="1" showInDefault="1" showInWebsite="0" showInStore="0">

```

Comment:
Try to using sort order values like 10, 20, 30, etc. (then you can in fast way put new field without changing others sort order values).

Reply:
@rafdol 

> Try to using sort order values like 10, 20, 30, etc. (then you can in fast way put new field without changing others sort order values).

sortOrder numbering fixed!